File: Exercises/JavaScript/JavaScript_Basics/Objects/ex5.js
// Before running any code, determine what difference there will be in the output of the two code snippets below (if any).

// snippet 1

let ocean = {};
let prefix = 'Indian';

ocean.prefix = 'Pacific';

console.log(ocean); // ?

// snippet 2

ocean = {};
prefix = 'Indian';

ocean[prefix] = 'Pacific';

console.log(ocean); // ?

// first snippet will print { prefix: 'Pacific' }
// second snippet will print { Indian: 'Pacific' }

File: Exercises/Python/Python_Basics/5_Functions/ex5.py
'''
Without running the following code, determine what it will print:
    # <function multiples_of_three at 0x32210...>
'''

def multiples_of_three():
    divisor = 1

    for dividend in range(3, 31, 3):
        print(f'{dividend} / {divisor} = 3')
        divisor += 1

print(multiples_of_three)
